payments-core-testing/src/main/java/com/stripe/android/testing/AbsFakeStripeRepository.kt

```diff
@@ -54,61 +54,61 @@ internal abstract class AbsFakeStripeRepository : StripeRepository() {
         options: ApiRequest.Options,
         expandFields: List<String>
     ): PaymentIntent? {
-        return null
+        TODO("Not yet implemented")
```

Comment:
Throwing an exception instead of returning a plausible value so that we’re forced to implement the methods that we rely on in tests.

Nothing seems to have broken following this 🤷